source/common/http/http2/conn_pool.cc

```diff
@@ -213,11 +224,15 @@ void ConnPoolImpl::onStreamReset(ActiveClient& client, Http::StreamResetReason r
   } else if (reason == StreamResetReason::RemoteReset) {
     host_->cluster().stats().upstream_rq_rx_reset_.inc();
   }
+
+  //Start the idle timer
+  client.startIdleTimer();
```

Comment:
Is the entire connection idle at this point?  Can there be other active streams?